src/store/actionCreators/saveSets.ts

```diff
@@ -230,6 +231,16 @@ export const deleteUserSets = (
   }
 };
 
+export const createQueryInCohortBuilder = (
+  setInfo: SetInfo,
+): ThunkAction<void, RootState, null, SetsActionTypes> => async (dispatch) => {
+  try {
+    dispatch(requestCreateQueryInCohort(setInfo));
+  } catch (e) {
+    console.error(e.message);
```

Comment:
what happen when it fails?

Reply:
thats just wrong on my part. it should not be inside a try catch